MAVProxy/mavproxy.py

```diff
@@ -1103,6 +1103,11 @@ def quit_handler(signum = None, frame = None):
     # call this early so that logdir is setup based on --aircraft
     (mpstate.status.logdir, logpath_telem, logpath_telem_raw) = log_paths()
 
+    for module in opts.load_module:
+        modlist = module.split(',')
+        for mod in modlist:
+            process_stdin('module load %s' % (mod))
```

Comment:
The left over `()` enclosing `mod` was an accidental left over from my PR dev... I can remove it there or since you are moving the logic it could be done here (or optionally not at all!)